src/pyflexplot/cli/main.py

```diff
@@ -147,6 +147,11 @@ def main(
     except FileNotFoundError:
         log(err="Error merging shape files.")
 
+    for path in redundant_shape_files:
```

Comment:
You can nest this into an else block otherwise the program is just going to crash here in case of FileNotFoundError

Reply:
Fixed it.